lib/main.dart

```diff
@@ -1,8 +1,21 @@
+import 'package:firebase_core/firebase_core.dart';
+import 'package:firebase_crashlytics/firebase_crashlytics.dart';
+import 'package:fitness_app/firebase_options/firebase_options.dart';
 import 'package:fitness_app/src/app.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter_dotenv/flutter_dotenv.dart' as dot_env;
 
-void main() {
-  runApp(
-    const MyApp(),
+Future main() async {
+  WidgetsFlutterBinding.ensureInitialized();
+  await Firebase.initializeApp(
+    options: DefaultFirebaseOptions.currentPlatform,
   );
+  
+  await dot_env.dotenv.load(
+    fileName: "assets/.env",
+  );
+  FlutterError.onError = FirebaseCrashlytics.instance.recordFlutterFatalError;
+  FirebaseCrashlytics.instance.crash();
+  
```

Comment:
What is the mean of the line 18? 
And reference boilerplate about calling Firebase Crashlytics in main function

Reply:
Line 18 mean create a crash intentionally to save the necessary information when testing.

To use Firebase Crashlytics, I used FlutterError.onError = FirebaseCrashlytics.instance.recordFlutterFatalError; to link my Flutter app's error handling to Firebase Crashlytic.

Reply:
line 18 is used for testing but you still committed

Reply:
follow the boilerplate to call Firebase Crashlytics